src/screens/LoadingAddresses.js

```diff
@@ -61,9 +61,15 @@ class LoadingAddresses extends React.Component {
     }
     // To prevent only a blink in this screen when user loads the addresses really fast
     // I set that the user will see this screen at least for 2 seconds
-    setTimeout(() => {
+    const timeoutId = setTimeout(() => {
       this.setState({ canRedirect: true });
     }, 2000);
+    this.setState({ timeoutId });
```

Comment:
Maybe this `timeoutId` could be `this.timeoutId` because it's not being used to render anything, only to be used in the class. What do you think?

Reply:
Done